src/app/components/nav/simple-header/simple-header.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="sim-header padding-lr-header">
+  <nav>
+    <div class="nav-wrapper">
+      <a href="javascript:void(0)" (click)="menuExpander()" class="button-collapse sim-header-link"><i class="fa fa-bars"></i></a>
+      <ul class="left ">
+        <li class="text-med-gray"><a class="waves-effect waves-dark dark-link ev-header-overflow dashboard-evailai-logo" routerLink="/"> <span class="hide-on-small-only"><img src="assets/images/evalai-logo.png"></span></a></li>
+        <li class="text-med-gray"><a class="waves-effect waves-dark dark-link main-header-link" routerLink="/challenges">All Challenges</a></li>
+        <li class="text-med-gray"><a class="waves-effect waves-dark dark-link main-header-link" href="https://evalai-forum.cloudcv.org/" target="_blank">Forum</a></li>
+      </ul>
+      <ul *ngIf="authService.isAuth" class="right">
+        <li class="text-med-gray"><a class="waves-effect waves-dark dark-link ev-header-overflow" routerLink="/profile"> <span class="hide-on-small-only" ><span class="text-light-black">Hi</span>{{' ' + user.username}} </span></a></li>
+        <li>
+          <div class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" (click)="logOut()">Logout</div>
+        </li>
+      </ul>
+      <ul *ngIf="!authService.isAuth" class="right">
+        <li class="text-med-gray"><a class="waves-effect waves-dark dark-link ev-header-overflow" routerLink="/auth/signup"> <span class="hide-on-small-only"> Sign Up </span></a></li>
+        <li>
+          <a class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" routerLink="/auth/login">Log In</a>
+        </li>
+      </ul>
+      <ul class="side-nav" style="width:200px; transform:translateX(0px);" (click)="menuExpander()" *ngIf="!isMenuExpanded">
```

Comment:
Move `style="width:200px; transform:translateX(0px);"` to `.scss`. (Also, at other places.)

Reply:
> > @lunayach it's working fine on my machine.
> > ![Screenshot from 2019-07-21 03-02-01](https://user-images.githubusercontent.com/17106489/61588121-f9a59080-ab63-11e9-8a18-06bffe78afa9.png)
> 
> Yes, re-building it helped. But as you might see, there is something wrong in the Home-page UI. Would this get fixed in #140?

yes for sure.